spec/knapsack_pro/slow_test_file_determiner_spec.rb

```diff
@@ -67,7 +67,7 @@
 
     context 'when json report does not exist' do
       it do
-        expect { subject }.to raise_error(RuntimeError, 'Report with slow test files was not generated yet. If you have enabled split by test cases https://github.com/KnapsackPro/knapsack_pro-ruby#split-test-files-by-test-cases and you see this error it means that your tests accidentally cleaned up tmp/knapsack_pro directory. Please do not remove this directory during tests runtime!')
+        expect { subject }.to raise_error(RuntimeError, 'Report with slow test files was not generated yet. If you have enabled split by test cases https://github.com/KnapsackPro/knapsack_pro-ruby#split-test-files-by-test-cases and you see this error it means that your tests accidentally cleaned up .knapsack_pro directory. Please do not remove this directory during tests runtime!')
```

Comment:
Suggestions for the error msg:

"_The_ report with slow test files"

I would surround the url with the parens so that it doesn't disrupt the sentence.

"_the_ .knapsack_pro directory"